test/data/BUILD

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (C) 2022 Vaticle
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+load("@vaticle_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
+
+filegroup(
+    name = "query-string-files",
+    srcs = [
+        "data_string.tql",
+        "schema_string.tql",
+        "query_string.tql",
+    ],
+    visibility = ["//test:__pkg__"],
+)
+
+checkstyle_test(
+    name = "checkstyle",
+    include = ["BUILD"],
+    exclude = glob(["*.tql"]),
+    license_type = "agpl",
+)
```

Comment:
Keeping these files in-line with the code is excessive and makes the tests harder to read. I've moved them into their own directory for clarity & demarcation.